src/widgets/index/index.ts

```diff
@@ -264,16 +264,51 @@ const index = (props: IndexProps): Index => {
           _uiState: localUiState,
         });
 
+        localWidgets.forEach(widget => {
```

Comment:
Is it normal to do it on `localWidgets` while we execute the `init` method only on `widgets` just after ?

Reply:
Good catch.

Reply:
7ebd868b3544439590abf57e04a10ffbbd50ff03